hiba-grl.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2022 The HIBA Authors
+ *
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+#include <inttypes.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <unistd.h>
+
+#define HIBA_INTERNAL
+
+#include "log.h"
+#include "hiba.h"
+#include "misc.h"
+#include "openbsd-compat/bsd-misc.h"
+#include "util.h"
+
+void
+usage() {
+	fprintf(stderr, "usage:\n");
+	fprintf(stderr, "  Revoke grants:\n");
+	fprintf(stderr, "    hiba-grl -f <file> -r -s <serial> [<id1> <id2> ...]\n");
```

Comment:
Do these commands really support multiple serials, as the usage string suggests?